src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptGoToSymbolService.cs

```diff
@@ -8,11 +8,15 @@
 using Microsoft.CodeAnalysis.GoToDefinition;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Navigation;
+using Microsoft.CodeAnalysis.Text;
+using System.Threading;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
 {
-    [ExportLanguageService(typeof(IGoToSymbolService), InternalLanguageNames.TypeScript), Shared]
-    internal sealed class VSTypeScriptGoToSymbolService : IGoToSymbolService
+    [ExportLanguageService(typeof(IAsyncGoToDefinitionService), InternalLanguageNames.TypeScript), Shared]
+    internal sealed class VSTypeScriptGoToSymbolService : IAsyncGoToDefinitionService
```

Comment:
TS internal side change of our EA layer.  the external side of course stays unchanged.

Reply:
🎉 hurray for external access